MYabrv.bib

```diff
@@ -481,35 +494,50 @@ @String{SchlossDagstuhl
  	@String{Dagstuhl = "Dagstuhl, Germany"}
 
 @String{SEI = "Software Engineering Institute"}
- 	@String{Pittsburgh = "Pittsburgh, PA, USA"}
- 	
- 	@String{Springer = "Springer"}
-    @String{BerlinHeidelberg = "Berlin, Heidelberg, Germany"}
-    % Sprinter International Publishing AG (old: Birkhäuser)
-    @String{Basel = "Basel, Switzerland"}
-    @String{Cham = "Cham, Switzerland"}
-    @String{Denver = "Denver, CO, USA"} % also for ACM
-    @String{Dordrecht = "Dordrecht, The Netherlands"}
-    @String{London = "London, UK"}
-    @String{Secaucus = "Secaucus, NJ, USA"}
-    @String{Wiesbaden = "Wiesbaden, Germany"}
-	@String{Leicester = "Leicester, United Kingdom"}
+
+@String{Springer = "Springer"}
+@String{BerlinHeidelberg = "Berlin, Heidelberg, Germany"}
+% Sprinter International Publishing AG (old: Birkhäuser)
+@String{Basel = "Basel, Switzerland"}
+@String{Cham = "Cham, Switzerland"}
+@String{Denver = "Denver, CO, USA"} % also for ACM
+@String{Dordrecht = "Dordrecht, The Netherlands"}
+@String{London = "London, UK"}
+@String{Secaucus = "Secaucus, NJ, USA"}
+@String{Wiesbaden = "Wiesbaden, Germany"}
+@String{Leicester = "Leicester, UK"}
+
+@String{TaylorFrancis = "Taylor \& Francis"}
   
+@String{Abingdon = "Abingdon, United Kingdown"}
 @String{Paris = "Paris, France"}
-@String{Paderborn = "Paderborn, Germany"}
 
-@String{TaylorFrancis = "Taylor \& Francis"}
-    @String{Abingdon = "Abingdon, United Kingdown"}
+@String{arXiv = "Cornell University Library"}
 
-@String{UniMagdeburg = "University of Magdeburg"}
-	@String{Magdeburg = "Magdeburg, Germany"}
+@String{UniAustin = "University of Texas at Austin, Department of Computer Science"}
+@String{HUBerlin = "Humboldt-Universität zu Berlin"}
 @String{TUBraunschweig = "TU Braunschweig"}
-	@String{Braunschweig = "Braunschweig, Germany"}
+@String{UniBrussel = "University of Brussel"}
+@String{CMU = "Carnegie Mellon University"}
+	@String{Pittsburgh = "Pittsburgh, PA, USA"}
```

Comment:
Why is this the only variably with indent. Shouldn't the city be moved down to the other US cities.

Reply:
Ah I see lol. For consistency I'd prefer moving it to the other cities but I accept both.